ironfish-rust-nodejs/tests/demo.test.slow.ts

```diff
@@ -58,7 +58,7 @@ describe('Demonstrate the Sapling API', () => {
   it(`Should create a miner's fee transaction`, () => {
     const key = generateKey()
 
-    const transaction = new Transaction(key.spendingKey)
+    const transaction = new Transaction(key.spendingKey, 2)
```

Comment:
Similar comment as the one below: would it make sense to use a `LATEST_TRANSACTION_VERSION` (or similar) for use in tests?